app/containers/SearchBox/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useCallback, useState } from 'react';
+import { StyleSheet, TextInputProps, View } from 'react-native';
+
+import { useTheme } from '../../theme';
+import I18n from '../../i18n';
+import FormTextInput from '../TextInput/FormTextInput';
+
+const styles = StyleSheet.create({
+	inputContainer: { margin: 16 }
+});
+
+const SearchBox = ({ onChangeText, onSubmitEditing, testID }: TextInputProps): JSX.Element => {
+	const { colors, theme } = useTheme();
+	const [text, setText] = useState('');
+
+	const internalOnChangeText = useCallback(value => {
+		setText(value);
+		onChangeText?.(value);
+	}, []);
+
+	return (
+		<View testID='searchbox'>
+			<FormTextInput
+				autoCapitalize='none'
+				autoCorrect={false}
+				blurOnSubmit
+				placeholder={I18n.t('Search')}
+				placeholderTextColor={colors.auxiliaryTintColor}
+				returnKeyType='search'
+				underlineColorAndroid='transparent'
+				containerStyle={styles.inputContainer}
+				onChangeText={internalOnChangeText}
+				onSubmitEditing={onSubmitEditing}
```

Comment:
If you're not overwriting, you can just remove it.
Same for testID.
It's adding unnecessary complexity.

Reply:
I have to pass this props, because as we don't pass `...props` at the component anymore, if I don't specify it on the search box component, we can't use it on the `TextInput` .
E.g Directory View. If I remove the `onSubmitEditing ` from here, the search button don't work.